src/rdf4cpp.hpp

```diff
@@ -0,0 +1,24 @@
+#ifndef RDF4CPP_RDF4CPP_HPP
+#define RDF4CPP_RDF4CPP_HPP
+
+#include <rdf4cpp/ClosedNamespace.hpp>
+#include <rdf4cpp/Dataset.hpp>
+#include <rdf4cpp/Namespace.hpp>
+#include <rdf4cpp/Node.hpp>
+#include <rdf4cpp/namespaces.hpp>
+#include <rdf4cpp/bnode_mngt/NodeGenerator.hpp>
+#include <rdf4cpp/bnode_mngt/NodeScope.hpp>
+#include <rdf4cpp/parser/IStreamQuadIterator.hpp>
+#include <rdf4cpp/parser/RDFFileParser.hpp>
+#include <rdf4cpp/version.hpp>
+#include <rdf4cpp/IRIFactory.hpp>
+#include <rdf4cpp/writer/SerializationState.hpp>
+
```

Comment:
add 
```
#include <rdf4cpp/storage/reference_node_storage/UnsyncReferenceNodeStorage.hpp>
```

Reply:
I think you meant `SyncReferenceNodeStorage`